### DISCUSSION TITLE
Voice Cloning for Urdu Language

### DISCUSSION THREAD

**Discussion by umairafzal9245** [coqui-ai/TTS#2461] [General Q&A]
I have trained the yourtts model along with resnet speaker encoder using transfer learning on my dataset consisting of 195 speakers. the audio generated are quite good but the model is not cloning the voice using 5 to 10 seconds of reference speech. can you please tell me do we need to train the vocoder also for voice cloning? here are my results of training
![image](https://user-images.githubusercontent.com/74504006/227975005-afd90671-27f0-46ae-a1a3-22f034302198.png)
what can be the possible issue?

**Comment by erogol** [comment]
You don't need to train vocoder. YourTTS has a vocoder. 
How was the quality? Would you share some samples? Just curious.

**Reply by umairafzal9245** [reply]
[Documents.zip](https://github.com/coqui-ai/TTS/files/11139577/Documents.zip)
Here are some voice samples. I dont know why the sound is bit robotic?

**Reply by usmanahmedraza1** [reply]
@umairafzal9245 any update on your experiment about voice cloning in Urdu?

**Reply by alyan-ahmad** [reply]
@umairafzal9245 did you succeed in cloning voice in Urdu? I have some quires can you help?

**Reply by segmentationFaults** [reply]
how do you du transfer learning?

**Reply by umairafzal9245** [reply]
i did the finetuning of model on my dataset.

**Reply by Kashmalajaved66** [reply]
@umairafzal9245 did you get the achual voice?

**Reply by umairafzal9245** [reply]
@Kashmalajaved66 yes

**Reply by Kashmalajaved66** [reply]
@umairafzal9245  can you help me ?

**Reply by umairafzal9245** [reply]
umairafzal92786@gmail.com
@Kashmalajaved66

**Reply by Kashmalajaved66** [reply]
Can you show me the final results?

**Reply by Kashmalajaved66** [reply]
Did you use VITS urdu pretrained model for finetuning?

**Comment by AbdullahMustafa040** [comment]
Hi i can help

**Comment by usmanahmedraza1** [comment]
Great

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Kashmalajaved66** [comment]
Hi , I need help to train TTS. Trying to train/finetune TTS for a specific vioce. Can anyone show me the final results of finetuning

**Reply by JahangirMoro** [reply]
can you please tell me how you trained the urdu tts?
